Library/Formula/disco.rb

```diff
@@ -13,25 +13,62 @@ class Disco < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Maybe we should promote this to always requiring a Tigerbrew python, given the age of even Snow Leopard's?
```suggestion
  depends_on :python
```

Reply:
That's my hypothesis too, based on the content of the error message. I'll test a build tonight and see whether I can reproduce this.

Reply:
Thank you!